dashboard/app/views/scripts/index.html.haml

```diff
@@ -18,4 +18,5 @@
           - if script_level
             %td= link_to 'Play', script_level_path(script, script_level)
           %td= link_to(t('crud.edit'), edit_script_path(script))
-          %td= link_to t('crud.destroy'), script, method: :delete, data: { confirm: t('crud.confirm') }
+          %td= link_to t('crud.destroy'), script, method: :delete, |
```

Comment:
Don't use the `|` character to split up lines. Wrap on commas or extract code into helper.